tests/crawl/test_crawl.py

```
@@ -0,0 +1,33 @@
+import pytest
+from rich import print as rprint
+
+
+import pytest
```

Ruff (0.8.2)

Redefinition of unused `pytest` from line 1

Remove definition: `pytest`

(F811)